### ISSUE TITLE
Better Warning messages for `pueue remove`

### ISSUE THREAD

**Issue by dhra** [Nukesor/pueue#108]
**Is your feature request related to a problem? Please describe.**
There are 10 tasks in queue and we want to remove from 5 till 8 or removing specify tasks: 2,9 

**Describe the solution you'd like**
```bash
pueue remove 5..8
pueue remove 2,9
```
or
```bash
pueue remove 5-8 2,9
```

**Comment by dhra** [comment]
A warning message for removing a bunch of tasks maybe useful:

```bash
$ pueue remove 5-8
you are trying to remove:
task5, task6, task7, task8

Are you sure to remove taks?[Y/n]
```

And an error message for removing a task which doesn't exist:

```bash
$ pueue remove 1111
error: `1111` doesn't exist in the list
```

**Comment by Nukesor** [comment]
1. 
```
pueue remove 5..8
pueue remove 2,9
```

This behavior is already implemented in bash/zsh and most other shells. Simply use `{5..8}` instead.

With this you can do stuff like this `pueue log {0..1} 4`

2. 
Warning messages can indeed be useful. However I would like to make them conditional (disable them via config).

3.
That should actually be already the case. I consider this a bug.

**Comment by Nukesor** [comment]
3. Has been fixed in b578daa866790d1a52c5817428384b172afe8e35

**Comment by Nukesor** [comment]
The implementation of warning messages is tracked in the linked issue.

This issue will be marked as resolved
